src/collections/_documentation/performance/distributed-tracing.md

```diff
@@ -12,105 +12,325 @@ Sentry's Performance features are currently in beta. For more details about acce
 
+### Why Tracing?
+
+Applications typically consist of interconnected components, which are also called services. As an example, let's look at a modern web application, composed of the following components, separated by network boundaries:
+
+- Front End (Single-Page Application)
+- Backend (REST API)
+- Task Queue
+- Database Server
+- Cron Job Scheduler
+
+Each of these components may be written in a different language on a different platform. Each can be instrumented individually using a Sentry SDK to capture error data or crash reports, but that instrumentation doesn't provide the full picture, as each piece is considered separately. Tracing allows you to tie all of the data together. 
+
+In our example web application, tracing means being able to follow a request from the front end to the backend and back, pulling in data from any background tasks or notification jobs that request creates. Not only does this allow you to correlate Sentry error reports, to see how an error in one service may have propagated to another, but it also allows you to gain stronger insights into which services may be having a negative impact on your application's overall performance.
+
+Before learning how to enable tracing in your application, it helps to understand a few key terms and how they relate to one another.
+
+### Traces, Transactions, and Spans
+
+A **trace** represents the record of the entire operation you want to measure or track - like page load, an instance of a user completing some action in your application, or a cron job in your backend. When a trace includes work in multiple services, such as those listed above, it's called a **distributed trace**, because the trace is distributed across those services. To recap, a trace is a record of an operation; a distributed trace is a record of an operation that's distributed across services.
+
+Each trace consists of one or more tree-like structures called **transactions**, the nodes of which are called **spans**. In most cases, each transaction represents a single instance of a service being called, and each span within that transaction represents that service performing a single unit of work, whether calling a function within that service or making a call to a different service.
+
+Because a transaction has a tree structure, top-level spans can themselves be broken down into smaller spans, mirroring the way that one function may call a number of other, smaller functions; this is expressed using the parent-child metaphor, so that every span may be the **parent span** to multiple other spans. Further, since all trees must have a single root, one span always represents the transaction itself, and all other spans are descendants of that root span.
```

Comment:
```suggestion
Because a transaction has a tree structure, top-level spans can themselves be broken down into smaller spans, mirroring the way that one function may call a number of other, smaller functions; this is expressed using the parent-child metaphor, so that every span may be the **parent span** to multiple other spans. Further, since all trees must have a single root, this one span always represents the transaction itself, and all other spans are descendants of that root span.
```

Reply:
I don't think this works, because we haven't introduced a particular span yet - only made statements about spans in general.

However, perhaps it's clearer to say, "Further, since all trees must have a single root, one span in a transaction always represents the transaction itself, with all other spans in the transaction descending from that root span."